### ISSUE TITLE
NEAR Social Notifications link to raw Wiki/Blog post types instead of the formatted versions within the blog widget

### ISSUE THREAD

**Issue by ori-near** [NEAR-DevHub/neardevhub-bos#505]
**Describe the bug**
Since introducing the blog post type, I noticed that they showed up in my near.social notifications. However, when I click on them – they point me to the [raw post](https://near.social/devgovgigs.near/widget/gigs-board.pages.Post?id=2029) instead of to the [formatted versions](https://near.social/devgovgigs.near/widget/app?page=blog&id=2029) within the blog widget.

**To Reproduce**
1. Go to https://near.social/mob.near/widget/NotificationFeed
2. Click on any notification related to editing a blog post type (you may need to edit one first)
3. Click View DevHub post
4. See raw version.

**Expected behavior**
Ideally we should not make the raw version of blog widget post types discoverable. Clicking on the notification should take users directly to the formatted blog widget version. But also if somebody goes to the raw version, it would be nice to automatically redirect them. 

**Screenshots**

[Unformatted Raw Version](https://near.social/devgovgigs.near/widget/gigs-board.pages.Post?id=2029)
![Screenshot 2023-11-22 at 11 23 51 AM](https://github.com/near/neardevhub-widgets/assets/110252255/bde9bf59-0d23-4ec6-9e3c-04ab760c4f2b)

[Formatted Blog Widget Version](https://near.social/devgovgigs.near/widget/app?page=blog&id=2029)
![Screenshot 2023-11-22 at 11 24 32 AM](https://github.com/near/neardevhub-widgets/assets/110252255/bbd4a670-ad10-416d-a4c6-fd980aaacbc6)

**Desktop (please complete the following information):**
 - OS: OS
 - Browser: Safari
 - Version: Version 17.0 (19616.1.27.211.1)

**Comment by Megha-Dev-19** [comment]
@ailisp who controls the notifications link from near org?

**Comment by ailisp** [comment]
@Megha-Dev-19 The notification logic for Devhub notifications is on our side and referenced on near.org. 

However it's accidentally dropped in dbfeaff821fbcea41cdf9d0549461d888e11fac2, let's recover these files and fix the issue:

```
git checkout dbfeaff821fbcea41cdf9d0549461d888e11fac2^ -- src/DevGov/Notification/Item/Left.jsx
git checkout dbfeaff821fbcea41cdf9d0549461d888e11fac2^ -- src/DevGov/Notification/Item/Right.jsx
```

**Comment by elliotBraem** [comment]
As discussed, this will require a contract change. This issue has been [created](https://github.com/near/neardevhub-contract/issues/73)

**Comment by Tguntenaar** [comment]
I will pick this up and work on the contract side to create an postType blog.

**Comment by Tguntenaar** [comment]
This issue can now be reviewed, PR [#81](https://github.com/NEAR-DevHub/neardevhub-contract/pull/81) must be merged first, before proceeding with #579 .

**Comment by Tguntenaar** [comment]
Closing this, will be fixed by the blog enhancements milestone
